src/unity/Assets/Crest/Materials/Ocean.mat

```diff
@@ -74,7 +74,7 @@ Material:
         m_Scale: {x: 1, y: 1}
         m_Offset: {x: 0, y: 0}
     m_Floats:
-    - _ApplyFlowToNormals: 0
```

Comment:
should be off by default on the main ocean mat

Reply:
isnt't that turned off?